include/wx/generic/grid.h

```diff
@@ -2816,6 +2825,11 @@ class WXDLLIMPEXP_CORE wxGrid : public wxScrolledCanvas
     // setting m_isDragging to true
     wxPoint m_startDragPos;
 
+public:
```

Comment:
I understand this needs to be public because it's used by `CheckDoDragScroll()`, but I don't understand why don't we make this function a member function instead -- this is much more idiomatic in C++ than calling `CheckDoDragScroll(this)` (and never passing it anything else). Is there something I'm missing here?

Reply:
Yes.